massgov/mayflower
# Feature/patternlab/dp 15223  remove sticky toc from print

## Description
Remove the sticky TOC from printed pages

## Related Issue / Ticket

- [DP-15223](https://jira.mass.gov/browse/DP-15223)

## Steps to Test

1. Go to https://mayflower.digital.mass.gov/b/feature/patternlab/DP-15223--remove-sticky-toc-from-print/patterns/05-pages-guide/05-pages-guide.html

2. Scroll down until the sticky TOC is visible

3. print the page to see that the sticky TOC is not visible on the printed page 

## Screenshots
<img width="526" alt="Screen Shot 2019-08-06 at 10 48 50 AM" src="https://user-images.githubusercontent.com/18662734/62555066-e1a66000-b837-11e9-8e7a-dc9e94ce6639.png">
<img width="517" alt="Screen Shot 2019-08-06 at 10 48 59 AM" src="https://user-images.githubusercontent.com/18662734/62555073-e539e700-b837-11e9-9816-976886137f0d.png">
